app/styles/_shame.scss

```diff
@@ -64,15 +64,15 @@
 // Descriptions > @todo: replace with label and p
 .au-c-description-label {
   @include au-font-size($au-h6);
+  padding: 0.8rem 0;
```

Comment:
I would not add this padding like this, this makes the component look a bit skewed, with more margin on the top and the bottom of the card then on the sides.
I would just remove this padding altogether, and maybe increase the padding of the grid a bit.